### ISSUE TITLE
Feature Request: Improved dashboard with pinnable prompts. Mockup Included. 

### ISSUE THREAD

**Issue by fmotion1** [gitbito/bitoai#175]
Hi, I am really loving Bito. I discovered it earlier the other day and signed up for the `10X Developer Plan` today. 

While the VSCode add-on is great, I think the dashboard could use a lot of work. 

1. Having all your custom prompts and templates buried away into the lower right is kind of frustrating. 
2. The input box with `/` being used to access templates is nice, but I still feel my prompts are kind of buried. 
3. IMO, it would be a much better UI/UX decision to have them actually on the dashboard as buttons that can be configured.
4. 90% of the dashboard is just introductory text and links to documentation. It would be absolutely killer if users could customize this page to their own personal liking. I.E. Placing commonly used prompts, etc. 

I spent about an hour tonight throwing a mock-up together. It's by no means perfect but it kind of demonstrates what I mean.

![Bito-BeforeAfter](https://github.com/gitbito/bitoai/assets/33441569/24b51a41-bd92-4cff-8a33-081a3f3ec1b9)

TL;DR Let us customize the dashboard!

Thanks for any consideration.

**Comment by bitoandy** [comment]
Hi @fmotion1 ,

Thanks for the detailed feedback and UI design, this is very helpful. The UI design looks great. Our product team is looking into this. We will keep you posted on the changes.
